ec/src/models/short_weierstrass/group.rs

```diff
@@ -634,6 +634,24 @@ impl<P: SWCurveConfig> ScalarMul for Projective<P> {
     }
 }
 
+#[cfg(feature = "specialization")]
+pub trait CustomMSM: SWCurveConfig {
+    fn msm_bigint(
+        bases: &[Affine<Self>],
+        bigints: &[<Self::ScalarField as PrimeField>::BigInt],
+    ) -> Projective<Self>;
+}
```

Comment:
We can also try adding this to method to `SWCurveConfig` directly, with a default impl calling the current variable-base MSM code.

Reply:
This was my initial idea! But this was what caused the recursive compile time loop problem. I just pushed the code here: https://github.com/achimcc/algebra/blob/add-method-to-trait/ec/src/models/short_weierstrass/mod.rs#L114 It builds but execution is extremely slow. That is why I changed the implementation to what it is now. I'd assume we would have to actually re-implement the method in `SWCurveConfig` trait to avoid this recursion?